common/lib/capa/capa/responsetypes.py

```diff
@@ -684,6 +687,7 @@ class ChoiceResponse(LoncapaResponse):
 
     """
 
+    human_name = _('Checkboxes')
```

Comment:
Do we need a TNL technical debt story to change our display of problem categories to use these human_names? The human_names are possibly now different from what Studio is showing in the Problem component button, correct?

If so, please create the technical debt story.